### PR TITLE
Switches most common visible_messages to look like emotes, inc. surgery.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
While observing I noticed a lot of messages being completely out of place now that we actually see visible messages, so here's the fix. I removed a bunch of spans and just bolded the name in the visible messages so that they match emotes and shit.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/hydroponics/trays/tray.dm [hunk 1]
```diff
@@ -454,9 +454,9 @@
 					var/amountToRemove = min((maxWaterLevel - waterlevel), RC.reagents.total_volume)
 					RC.reagents.remove_reagent("water", amountToRemove, 1)
 					waterlevel += amountToRemove
-					user.visible_message("[user] transfers [amountToRemove]u of water to the tray."," You transfer [amountToRemove]u of water to the tray.")
+					user.visible_message("<b>[user]</b> transfers some water to the tray.", "You transfer about [amountToRemove] units of water to the tray.")
 				else
-					to_chat(user, "This tray is full of water already.")
+					to_chat(user, SPAN_NOTICE("This tray is full of water already."))
```
Comment (Geevies) [inline code/modules/hydroponics/trays/tray.dm:459]:
```suggestion
					to_chat(user, SPAN_WARNING("This tray is full of water already."))
```

File: code/modules/surgery/organs_internal.dm [hunk 2]
```diff
@@ -65,15 +65,15 @@
 	if(istype(tool, /obj/item/stack/medical/advanced/bruise_pack))
 		tool_name = "regenerative membrane"
 	if(istype(tool, /obj/item/stack/medical/bruise_pack))
-		tool_name = "the bandaid"
+		tool_name = "some bandaids"
```
Comment (Geevies) [inline code/modules/surgery/organs_internal.dm:68]: gauze, innit?

**Comment by MarinaGryphon** [comment]
oh no this is going to conflict horribly with #8998

**Comment by MarinaGryphon** [comment]
actually span_bold has issues (it's not the whole message being bolded) so maybe just `<span class='bold'>` though i suppose it could be changed with all the other emotes in another PR

**Pull request merged**